static/src/stylesheets/module/commercial/_adverts-soulmates.scss

```diff
@@ -1,6 +1,6 @@
 .adverts--soulmates {
     @include mq($until: tablet) {
-        .adverts__row {
+        &:not(.adverts--manual) .adverts__row {
```

Comment:
When I look at this whole block I have a strong feeling that when you wrote this, only God and you understood what you were doing. And in a few weeks only God will know.